ror-shadowed-libs/build.gradle

```diff
@@ -57,6 +57,8 @@ dependencies {
     shadowCompile group: 'com.fasterxml.jackson.core',            name: 'jackson-databind',                       version: '2.13.5'
     shadowCompile group: 'org.yaml',                              name: 'snakeyaml',                              version: '2.2'
     shadowCompile group: 'com.jayway.jsonpath',                   name: 'json-path',                              version: '2.9.0'
+    //It's a transitive dependency for json-path, listed here to fix CVE. It can be removed after json-path update
```

Comment:
I wonder if we shouldn't exclude it from the json path to make sure our dependency is always picked. WDYT?